README.md

```diff
@@ -480,17 +496,17 @@ Benchmark 3: dune exec solutions/tak_par.exe 4 36 24 12
 ```
 
 Observe that there is super-linear speedup going from the sequential version to
-the 2 core version! Why?
+the two-core version! Why?
 
 
 #### Exercise ★★★★★
 
 Implement a parallel version of merge sort. It easy to implement a version that
-doesn't scale :-) If you use a list for holding the intermediate results, GC
+doesn't scale :-). If you use a list for holding the intermediate results, the GC
 impact will kill scalability. 
 
-You should use an array for holding the elements to be sorted. The observation
-is that during the merge step, the length of the merged result is exactly the
+You should use an [array](https://kcsrk.info/webman/manual/values.html#ss%3Avalues%3Aarray) for holding the elements to be sorted. We observed 
```

Comment:
We could link the official man page - https://v2.ocaml.org/api/Array.html. KC's page has the same source, but may not be updated for future released of OCaml.